src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -317,7 +317,7 @@ private async Task UpdateAdvertisingManifestAsync(WorkloadManifestInfo manifest,
                         // then try downloading the manifest's feature band
                         if (!(manifest.ManifestFeatureBand).Equals(_sdkFeatureBand))
                         {
-                            packagePath = await GetOnlinePackagePath(new SdkFeatureBand(manifest.ManifestFeatureBand), manifestId, includePreviews);
+                            packagePath = await GetOnlinePackagePath(new SdkFeatureBand(manifest.ManifestFeatureBand), manifestId, includePreviews);        // goes to failure message from here
```

Comment:
Instead of letting the exception fall through here, I think it would be better to catch it immediately (as is done on line 313).